scripts/MultiPlotting/navigation_toolbar.py

```diff
@@ -6,7 +6,7 @@
 class myToolbar(NavigationToolbar):
     # only display the buttons we need
```

Comment:
This is definitely something that needs to be comvered with testing, If anyone uses ``MultiPlotWindow`` in the future I forsee someone making a change that will bring back the ``home`` tool. Will you have time to implement tests before the release?

Reply:
in the maintenance period we are going to harmonise this with the workbench plotting. So it will disappear. When we have more time, I think the solution is to overwrite the functionality of the home button (didn't have much luck yesterday).

Reply:
In that case the current solution will have to do till then. Everything else looks fine, I couldn't find any other bugs. There is a lot of slowdown when multiple element lines are plotted but this is unavoidable with the number of lines involved.